test/uk/gov/hmrc/claimvatenrolmentfrontend/utils/helpers/TestConstants.scala

```diff
@@ -24,6 +24,7 @@ object TestConstants {
   val testJourneyId: String = UUID.randomUUID().toString
   val testContinueUrl: String = "/test"
   val testVatRegDate: LocalDate = LocalDate.parse("2021-01-01")
+  val testVatReturn: Boolean = true
```

Comment:
In the other contestants file this is "yes". I think the form is expecting yes rather than a boolean